sklearn/linear_model/tests/test_bayes.py

```diff
@@ -200,22 +202,18 @@ def test_toy_ard_object():
     assert_array_almost_equal(clf.predict(test), [1, 3, 4], 2)
 
 
-def test_ard_accuracy_on_easy_problem():
+@pytest.mark.parametrize('seed', range(100))
+def test_ard_accuracy_on_easy_problem(seed):
```

Comment:
The test should now run properly on many different seeds (there was a 7% failure rate before), and which a *much* higher precision